net-vpn/openvpn/openvpn-2.6.14.ebuild

```diff
@@ -0,0 +1,160 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools systemd linux-info tmpfiles
+
+DESCRIPTION="Robust and highly flexible tunneling application compatible with many OSes"
+HOMEPAGE="https://openvpn.net"
+
+if [[ ${PV} == "9999" ]]; then
+	EGIT_REPO_URI="https://github.com/OpenVPN/${PN}.git"
+	inherit git-r3
+else
+	SRC_URI="https://build.openvpn.net/downloads/releases/${P}.tar.gz"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~loong ~mips ~ppc ~ppc64 ~riscv ~sparc ~x86"
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="dco down-root examples inotify iproute2 +lz4 +lzo mbedtls pam pkcs11 +plugins selinux systemd test"
+RESTRICT="!test? ( test )"
+REQUIRED_USE="
+	dco? ( !iproute2 )
+	pkcs11? ( !mbedtls )
+	!plugins? ( !inotify !pam !down-root )
```

Comment:
I don't think this is the most helpful way of writing this for users to understand.